src/components/NewWorkspaceModal.js

```diff
@@ -118,8 +118,8 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
     Ajax(signal).Groups.list().then(setAllGroups),
     !!cloneWorkspace && Ajax(signal).Workspaces.workspace(namespace, cloneWorkspace.workspace.name).checkBucketLocation(cloneWorkspace.workspace.googleProject, cloneWorkspace.workspace.bucketName)
       .then(({ location }) => {
-        // For current phased regionality release, we only allow US workspace buckets.
-        setBucketLocation(isUSLocation(location) ? location : defaultLocation)
+        // For current phased regionality release, we only allow US or NORTHAMERICA-NORTHEAST1 (Montreal) workspace buckets.
+        setBucketLocation(isUSLocation(location) || _.eq(montrealRegion, location) ? location : defaultLocation)
```

Comment:
If we are going to continue to slowly roll out locations, could we have a method like `isSupportedBucketLocation`? That method's implementation would change over time, but usages wouldn't care, and we wouldn't need to do things like expose specific locations outside of region-common.

Reply:
Thank you! This is **much** better